src/main/java/io/vertx/httpproxy/impl/HttpProxyImpl.java

```diff
@@ -69,6 +63,14 @@ public void handle(HttpServerRequest outboundRequest) {
     });
   }
 
+  private Future<WebSocket> resolveOriginWS(HttpServerRequest outboundRequest) {
+    return selector.apply(outboundRequest).flatMap(server -> {
+      WebSocketConnectOptions webSocketConnectOptions = new WebSocketConnectOptions();
```

Comment:
there is not opportunity to configure WebSocketConnectOptions to the origin.

Reply:
Related code has been fixed.